src/synthesized_insight/version.py

```diff
@@ -0,0 +1,5 @@
+# coding: utf-8
+# file generated by setuptools_scm
+# don't change, don't track in version control
+__version__ = version = '0.8.dev3+g8cdaa326.d20220622'
+__version_tuple__ = version_tuple = (0, 8, 'dev3', 'g8cdaa326.d20220622')
```

Comment:
You might want to add `src/synthesized_insight/version.py` to the `.gitignore` file in this repository to avoid this file being committed by the way.